.github/workflows/build.yml

```diff
@@ -0,0 +1,94 @@
+name: PR Decoration
+on:
+  push:
+    branches:
+      - develop
+      - production
+      - pre-production
+      - 'release/v*'  # Branches for the production release e.g. release/vx.x.x
+      - 'staging-v*'  # Branches for the pre-production release e.g. staging-vx.x.x
+      - '**'  # This will run on PR commits or any other branch
```

Comment:
does this mean we plan on sonar runs on every branch / commit on the SDK not selectively with the `ready-for-sonar-checks` label?